libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -897,6 +988,47 @@ def Delete(self, delete_disks: bool = False) -> None:
             self.FormatLogMessage(
                 'Could not find disk: {0:s}, skipping'.format(disk_name)))
 
+  def SetTags(self, new_tags: List[str]) -> None:
+    """Sets network tags for the compute instance.
+
+    Args:
+        new_tags (List[str]): A list of tags. Each tag must be 1-63
+            characters long, and comply with RFC1035.
+
+    Raises:
+      InvalidNameError: If the name of the snapshot does not
+          comply with RFC1035.
+    """
+
+    logger.info(
+        self.FormatLogMessage(', adding tags {0!s} to instance '
+            '{1:s}.'.format(new_tags, self.name)))
+    for tag in new_tags:
+      if not common.REGEX_DISK_NAME.match(tag):
+        raise errors.InvalidNameError(
+            'Network Tag {0:s} does not comply with {1:s}.'.format(
+                tag, common.REGEX_DISK_NAME.pattern), __name__)
+
+    get_operation = self.GetOperation()
+    tags_object = get_operation['tags']
```

Comment:
This is a dict rather than an object?

Reply:
I changed it to dict. I meant API object, but dict is consistent with the rest of the code.